### PR TITLE
New package: DuckStudio.GitHubView.Nightly version 2024.12.25.12494060383

### PR DESCRIPTION
**Pull request by DuckDuckStudio**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/202171)

### CODE REVIEW

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 10:51:22 AM: Description: A .NET application failed. Application: ghv.exe Path: C:\Program Files (x86)\GitHubView\ghv.exe Message: You must install .NET to run this application.                                                                                                                         App: C:\Program Files (x86)\GitHubView\ghv.exe                                                                          Architecture: x64                                                                                                       App host version: 8.0.11                                                                                                .NET location: Not found                                                                                                                                                                                                                        Learn more: https://aka.ms/dotnet/app-launch-failed  Download the .NET runtime: https://aka.ms/dotnet-core-applaunch?missing_runtime=true&arch=x64&rid=win-x64&os=win10&apphost_version=8.0.11

(Automated response - build 931.)

File: manifests/d/DuckStudio/GitHubView/Nightly/2024.12.25.12494060383/DuckStudio.GitHubView.Nightly.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Commands:
+- ghv
+Protocols:
+- https
+ReleaseDate: 2024-12-25
```
Comment (DuckDuckStudio) [inline manifests/d/DuckStudio/GitHubView/Nightly/2024.12.25.12494060383/DuckStudio.GitHubView.Nightly.installer.yaml:17]:
```suggestion
ReleaseDate: 2024-12-25
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.8
```

Try fix

File: manifests/d/DuckStudio/GitHubView/Nightly/2024.12.25.12494060383/DuckStudio.GitHubView.Nightly.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.8
```
Comment (DuckDuckStudio) [inline manifests/d/DuckStudio/GitHubView/Nightly/2024.12.25.12494060383/DuckStudio.GitHubView.Nightly.installer.yaml:20]:
```suggestion
  - PackageIdentifier: Microsoft.DotNet.Runtime.8
```

**Comment by DuckDuckStudio** [comment]
This seems to be a issue with dependencies:
image

**Comment by DuckDuckStudio** [comment]
aaa

**Comment by stephengillie** [comment]
Verify:
image
The VM isn't set up to display the characters.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**